core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -79,6 +79,36 @@ var ExplorationPlayerPage = function() {
     nextCardButton.click();
   };
 
+  this.clickSuggestChangesButton = function() {
+    waitFor.elementToBeClickable(suggestionPopupLink,
+      'Suggest changes button taking too long to appear');
+    suggestionPopupLink.click();
+  };
+
+  this.fillAndSubmitSuggestion = function(
+      suggestionTitle, suggestionDescription) {
+    var suggestionModal = element(
+      by.css('.protractor-test-exploration-suggestion-modal'));
+    waitFor.visibilityOf(suggestionModal,
+      'Suggesion Modal taking too long to appear');
```

Comment:
`Suggesion Modal is taking too long to appear.`

Reply:
Done!